contracts/query-queue/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "query-queue"
 version = "0.0.0"
 authors = ["Ethan Frey <ethanfrey@users.noreply.github.com>"]
```

Comment:
we need to fix the authors of `query-queue` contract. because it's made by @shiki-tak .

Reply:
Oh, it is made by me. Thank you for finding it! I will push a new PR to fix it.